quetz/tasks/mirror.py

```diff
@@ -317,10 +315,51 @@ def initial_sync_mirror(
                 f"in channel {channel_name}"
             )
             if repodata_fn == "repodata.json":
-                return
+                return None
 
-    channel = dao.get_channel(channel_name)
+    return {}
+
+
+def initial_sync_mirror(
```

Comment:
Why is this function called `initial`?  Does it not cover syncs after the first one?

Reply:
Yes, confused me as well :)